sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -148,6 +150,13 @@ Mono<Response<Void>> recognizeWithResponseInternal(CallMediaRecognizeOptions rec
                     dtmfOptionsInternal.setMaxTonesToCollect(dtmfRecognizeOptions.getMaxTonesToCollect());
                 }
 
+                if (dtmfRecognizeOptions.getStopTones() != null) {
+                    List<DtmfTone> dtmfTones = dtmfRecognizeOptions.getStopTones().stream()
+                                                .map(e -> translateDtmfToneToInternal(e))
```

Comment:
this can be changed to `.map(this::translateDtmfToneToInternal)`

Reply:
will change the style